task_11/build/index.html

```diff
@@ -0,0 +1,300 @@
+                        </tr>
+                    </table>
+                </div>
+            </section>
+
+            <section class="main-content__chose-languages">
+                <h2>Chose your languages pairings</h2>
+                <div class="main-content__chose-languages-input-wrapper">
+                    <div class="main-content__chose-languages-from">
+                        <p>
+                            From
+                        </p>
+                        <select id="lang-input-from" title="from">
+                            <option>English</option>
+                        </select>
+                    </div>
+                    <div class="main-content__chose-languages-to">
+                        <p>
+                            To
+                        </p>
+                        <select id="lang-input-to-1" title="to">
+                            <option>Arabic</option>
+                        </select>
+                        <label for="lang-input-to-2">
+                        </label>
+                        <select id="lang-input-to-2">
+                            <option>Kurdish</option>
+                        </select>
+                        <button type="button" class="main-content__button-delete-icon"><img src="img/icons/delete.svg">
+                        </button>
+                        <input type="button" value="+ ADD" class="main-content__chose-languages-to-add-button">
+                    </div>
+                </div>
+            </section>
+
+            <section class="main-content__select-tone">
+                <h2>Select your desired tone</h2>
+                <label for="friendly-tone">
+                    <input type="radio" id="friendly-tone" name="website-radio-button" required>
+                    <span class="main-content__select-tone-img-wrapper">
+                        <img src="img/icons/communication.svg">
+                        Friendly / Casual
+                    </span>
+                </label>
+                <label for="pro-tone">
+                    <input type="radio" id="pro-tone" name="website-radio-button" required>
+                    <span class="main-content__select-tone-img-wrapper">
+                    <img src="img/icons/suitcase.svg">
+                        <span>Corporate / Pro</span>
+                    </span>
+                </label>
+                <label for="media-tone">
+                    <input type="radio" id="media-tone" name="website-radio-button" required>
+                    <span class="main-content__select-tone-img-wrapper">
+                        <img src="img/icons/sound.svg">
+                        <span>Media / Formal</span>
+                    </span>
+                </label>
+            </section>
+
+            <section class="main-content__urgent-range">
+                <h2>Tell us how urgent this job is</h2>
+                <input type="range" id="urgent-range">
```

Comment:
You can remove focus outline effect from this element with outline: none

Reply:
But I thought that it's good from the point of accessibility, in case we can use 'tab' to navigate on the page. Or is this not true in this case?